packages/aurora-feed/test/feed.test.js

```diff
@@ -44,6 +44,25 @@ describe("feed", () => {
     expect(wrapper.containsMatchingElement(<span>{text}</span>)).toBe(true);
   });
 
+  it("can use delete to remove a card", () => {
+    const text = "Hey I will be deleted";
+    const wrapper = mount(<Feed persist={fake} />);
+    wrapper
+      .get(0)
+      .onSubmit(
+        EditorState.createWithContent(ContentState.createFromText(text))
+      );
+
+    // Run a sanity check to make sure we actually added something
+    expect(wrapper.containsMatchingElement(<span>{text}</span>)).toBe(true);
+
+    // Now let's remove it.
+    wrapper.find(".delete-button").simulate("click");
```

Comment:
how does it find that specific delete button? Does wrapper only wrap the card that was added?

Reply:
It doesn't, it only finds "a" delete button. But since it's a brand new feed with only one thing added, there should only be one card.